### PR TITLE
Add profile page of the current user

### PR DESCRIPTION
**Pull request by Xelowak**
I added the profile page of the current user.
For that, I first created a function to get chirps for a user. Then, I did the UI. 

Here is how it is displayed when you don't have a  PoP token yet:
Capture d’écran 2022-01-11 à 17 54 37

And when you have a token:
Capture d’écran 2022-01-11 à 17 25 48

I also disabled swipes in the social media navigation, to prevent weird interactions.

### CODE REVIEW

File: fe1-web/res/strings.ts [hunk 1]
```diff
@@ -52,6 +52,7 @@ const STRINGS = {
   social_media_navigation_tab_search: 'Search',
   social_media_navigation_tab_follows: 'My Follows',
   social_media_navigation_tab_profile: 'My Profile',
+  social_media_navigation_tab_user_profile: 'User profile',
```
Comment (jiabaow) [inline fe1-web/res/strings.ts:55]:
```suggestion
  social_media_navigation_tab_user_profile: 'User Profile',
```

File: fe1-web/store/reducers/__tests__/SocialReducer.test.ts [hunk 2]
```diff
@@ -120,62 +122,79 @@ const filledState4 = {
...
-
-test('selector should add the newer chirp after the second chirp', () => {
-  socialReduce(filledState2, addChirp(mockLaoId, chirp3));
-  expect(makeChirpsList().resultFunc(filledState3, mockLaoId))
-    .toEqual([chirp2, chirp3, chirp1]);
-});
-
-test('selector should return the newest chirp on top', () => {
-  socialReduce(filledState3, addChirp(mockLaoId, chirp4));
-  expect(makeChirpsList().resultFunc(filledState4, mockLaoId))
```
Comment (jiabaow) [inline fe1-web/store/reducers/__tests__/SocialReducer.test.ts:179]: I've also noticed the tests are not readable and modified the structure, in particular, separating the tests for reducer and the tests for selector... Maybe you can also separate these two. But I don't know which PR will be marged first😂.

**Review by jiabaow — APPROVED** [review]
LGTM

File: fe1-web/store/reducers/__tests__/SocialReducer.test.ts (hunk 2 above)
Comment (Xelowak) [reply fe1-web/store/reducers/__tests__/SocialReducer.test.ts:179]: Ok, I've separated them.

File: fe1-web/store/reducers/SocialReducer.ts [hunk 3]
```diff
@@ -175,6 +175,33 @@ export const makeChirpsList = () => createSelector(
...
+          userChirps.forEach(
+            (id: string) => allUserChirps.push(chirpList.byLaoId[laoId].byId[id]),
+          );
+          return allUserChirps;
+        }
+      }
+      return [];
+    },
+  );
+};
```
Comment (pierluca) [inline fe1-web/store/reducers/SocialReducer.ts:203]: I would move this to the respective reducer, for coherence with the other selectors who are all there.

**Review by pierluca — APPROVED** [review]
LGTM, just a minor remark that could even be put in its own PR, if you'd rather merge this fast.

File: fe1-web/store/reducers/SocialReducer.ts (hunk 3 above)
Comment (Xelowak) [reply fe1-web/store/reducers/SocialReducer.ts:203]: I'm not sure I understand where I should put this. Isn't it already at the right place ? `makeChirpsList()` is right above it.

Comment (pierluca) [reply fe1-web/store/reducers/SocialReducer.ts:203]: You're absolutely right, it's in the right place.

My brain had a segfault 😅 I was somehow convinced that this was in a React Component ... I don't even know how or why 🤷

**Comment by pierluca** [comment]
@Xelowak I'll let you resolve the conflicts and we can merge 👍

**Review by pierluca — APPROVED** [review]
LGTM !

**Pull request merged**